apps/protocol-api/src/main.ts

```diff
@@ -231,6 +232,11 @@ const permissions = shield(
       contribution_reporting_channel: or(hasToken, isAuthenticated),
       status: or(hasToken, isAuthenticated),
     },
+    GuildActivityType: {
```

Comment:
(Arbitrary comment area)

Yes, we should add a personal section, but I think we can do this in a fast follow ticket. As a user can still add custom types, and I think the personal will require a whole new back to front endpoint.